docs/how-to-build.md

```diff
@@ -88,11 +88,11 @@ This software is licensed under the Apache License version 2."
    with specific module name, like:
 
     ```shell
-    ./gradlew spark-connector:spark-connector-runtime:build
+    ./gradlew spark-connector:spark-runtime-3.4:build -PscalaVersion=2.12
```

Comment:
Isn't there a default scala version? For example, if `-PscalaVersion` is not specified, `scala 2.12` is used by default.

Reply:
ok. i see it in https://github.com/datastrato/gravitino/pull/3608.

but I still think there should be some extra description here.  for example, to explain the difference between  specifying  -`PscalaVersion` and not specifying.

Reply:
done